gst/tensor_filter/tensor_filter.c

```diff
@@ -502,7 +540,9 @@ gst_tensor_filter_set_property (GObject * object, guint prop_id,
         gchar **str_types;
 
         str_types = g_strsplit (g_value_get_string (value), ",", -1);
-        prop->input_meta.num_tensors = g_strv_length (str_types);
+        if (prop->input_meta.num_tensors != g_strv_length (str_types))
+          GST_ERROR ("The Number of Tensors is different.\
+           Num of INPUT != Num of TYPE\n");
```

Comment:
1. What about the initial num_tensors vaule? (always print GST_ERROR ?)
2. What if we are going to dynamically change?
3. Use GST_OBJECT_ERROR

Be careful, unlike our gst-launch or unittest_sink cases, an application may give {INPUTTYPE=NULL & INPUTNAME="blahblah"} and then give {INPUTYPE="whatsoever" & INPUTNAME="blahblah"}

Reply:
it's gonna be another issue: which property could decide the number of tensors in various cases.